src/common/CMakeLists.txt

```diff
@@ -100,6 +100,11 @@ set(common_srcs
   util.cc
   version.cc)
 
+if(WIN32)
+  list(APPEND common_srcs
+    dlfcn_win32.cc)
+endif()
```

Comment:
could you do something like
```cmake
if(WIN32)
  add_library(dlfcn_win32 STATIC dlfcn_win32.cc)
endif()
# ...
if(WIN32)
  list(APPEND ceph_common_deps dlfcn_win32)
endif()
```
?

Reply:
Sure. That makes sense, since there are cases in which you may not want to use ceph-common (e.g. crimson).